src/vmware/azext_vmware/commands.py

```diff
@@ -4,209 +4,97 @@
+        g.custom_command('add-identity-source', 'privatecloud_addidentitysource', deprecate_info=g.deprecate(redirect='az vmware private-cloud identity-source create', hide=True))
+        g.custom_command('deleteidentitysource', 'privatecloud_deleteidentitysource', deprecate_info=g.deprecate(redirect='az vmware private-cloud identity-source delete', hide=True))
+        g.custom_command('delete-identity-source', 'privatecloud_deleteidentitysource', deprecate_info=g.deprecate(redirect='az vmware private-cloud identity-source delete', hide=True))
         g.custom_command('add-cmk-encryption', 'privatecloud_addcmkencryption', deprecate_info=g.deprecate(redirect='az vmware private-cloud enable-cmk-encryption', hide=True))
         g.custom_command('delete-cmk-encryption', 'privatecloud_deletecmkenryption', deprecate_info=g.deprecate(redirect='az vmware private-cloud disable-cmk-encryption', hide=True))
         g.custom_command('enable-cmk-encryption', 'privatecloud_addcmkencryption')
         g.custom_command('disable-cmk-encryption', 'privatecloud_deletecmkenryption')
-        g.custom_command('rotate-vcenter-password', 'privatecloud_rotate_vcenter_password')
         g.custom_command('rotate-nsxt-password', 'privatecloud_rotate_nsxt_password')
 
-    with self.command_group('vmware private-cloud identity', vmware_sdk, client_factory=cf_vmware) as g:
+    with self.command_group('vmware private-cloud identity') as g:
         g.custom_command('assign', 'privatecloud_identity_assign')
         g.custom_command('remove', 'privatecloud_identity_remove')
         g.custom_show_command('show', 'privatecloud_identity_get')
 
-    with self.command_group('vmware cluster', vmware_sdk, client_factory=cf_vmware) as g:
-        g.custom_command('create', 'cluster_create')
-        g.custom_command('update', 'cluster_update')
-        g.custom_command('list', 'cluster_list')
-        g.custom_command('delete', 'cluster_delete')
-        g.custom_command('list-zones', 'cluster_list_zones')
-        g.custom_show_command('show', 'cluster_show')
-
-    with self.command_group('vmware authorization', vmware_sdk, client_factory=cf_vmware) as g:
-        g.custom_command('create', 'authorization_create')
-        g.custom_command('list', 'authorization_list')
-        g.custom_command('delete', 'authorization_delete')
-        g.custom_show_command('show', 'authorization_show')
-
-    with self.command_group('vmware hcx-enterprise-site', vmware_sdk, client_factory=cf_vmware) as g:
-        g.custom_command('create', 'hcxenterprisesite_create')
-        g.custom_command('list', 'hcxenterprisesite_list')
-        g.custom_command('delete', 'hcxenterprisesite_delete')
-        g.custom_show_command('show', 'hcxenterprisesite_show')
-
-    with self.command_group('vmware location', vmware_sdk, client_factory=cf_vmware) as g:
+    with self.command_group('vmware location') as g:

```

Comment:
Just out of curiosity, if the below commands are not migrated, why would the client factory be removed from the context?

Reply:
Because they use aaz commands in custom function internally, so there's no need to use client_factory which is designed for python SDK usage.